src/core/QCheck.mli

```diff
@@ -516,6 +516,8 @@ module Shrink : sig
   val char : char t (** @since 0.6 *)
 
   val int : int t
+  val int32 : int32 t
```

Comment:
would you mind adding a `@since NEXT_RELEASE` on both of these? (beware the warning 50, needs more whitespace)

Reply:
did it myself, nevermind. thanks for the work!